src/TestEnvironment.Docker/DockerApi/Abstractions/Services/IDockerImagesService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using TestEnvironment.Docker.DockerApi.Abstractions.Models;
+
+namespace TestEnvironment.Docker.DockerApi.Abstractions.Services
```

Comment:
I would prefer do not split namespaces into Abstractions and Internal. I think having only DockerApi would be enough.